packages/orbit-components/src/Itinerary/ItinerarySegment/context.js

```diff
@@ -0,0 +1,46 @@
+// @flow
+import * as React from "react";
+
+import type { Context, ProviderProps } from "./context";
+
+export const ItinerarySegmentContext: React.Context<Context> = React.createContext({
```

Comment:
i did not do that because of the flow:
`Cannot build a typed interface for this module. You should annotate the exports of this module with types. Cannot determine the type of this call expression. Please provide an annotation, e.g., by adding a type cast around this expression.Flow(signature-verification-failure)
`